### ISSUE TITLE
could not build wheels for pycuda and nvidia tensorrt version not found

### ISSUE THREAD

**Issue by chethanreddy1** [JunkyByte/easy_ViTPose#19]
i am trying to install requirements_gpu i did install PyTorch cuda before trying to install but I am getting this errors

**Comment by JunkyByte** [comment]
Hello could you share the error you are getting?

If the tensorrt or pycuda version is not found you might need to install nvidia-pyindex first, could you test this for me?
```bash
pip install nvidia-pyindex
pip install nvidia-tensorrt==8.4.3.1
```

**Comment by chethanreddy1** [comment]
![image](https://github.com/JunkyByte/easy_ViTPose/assets/95535293/be67896a-6929-4855-be37-9e14ac8a9895)

**Comment by JunkyByte** [comment]
Thanks, As a workaround we can use tensorrt 8.4.3.5, I tested it and it works fine (but the pypi package name is different)
try `pip install tensorrt==8.5.1.7`, could you try it?

**Comment by chethanreddy1** [comment]
nope i still get same error 
![image](https://github.com/JunkyByte/easy_ViTPose/assets/95535293/baafe895-0a1b-4182-8dc4-63e30aacb91c)

**Comment by JunkyByte** [comment]
What does nvidia-smi return?

**Comment by chethanreddy1** [comment]
![image](https://github.com/JunkyByte/easy_ViTPose/assets/95535293/cc3cc111-df0b-4610-b340-1ae4d1af91e5)

**Comment by JunkyByte** [comment]
I guess your cuda version is too recent to work with those versions. Try `pip install tensorrt==8.6.1` which is the oldest listed and see if it works for you..

**Comment by chethanreddy1** [comment]
yeah that worked but i still cant install pycuda, the same 'couldnt build wheels ' error

**Comment by chethanreddy1** [comment]
yeah building pycuda by myself worked

**Comment by JunkyByte** [comment]
Closing, feel free to reopen
